src/main/java/org/broadinstitute/hellbender/tools/sv/BafEvidence.java

```diff
@@ -0,0 +1,62 @@
+package org.broadinstitute.hellbender.tools.sv;
+
+import htsjdk.tribble.Feature;
+import org.broadinstitute.hellbender.utils.Utils;
+
+import java.util.Objects;
+
+public final class BafEvidence implements Feature {
+
+    final String sample;
+    final String contig;
+    final int position;
+    final double value;
+
+    public BafEvidence(final String sample, final String contig, final int position, final double value) {
+        Utils.nonNull(sample);
+        Utils.nonNull(contig);
+        this.sample = sample;
+        this.contig = contig;
+        this.position = position;
+        this.value = value;
+    }
+
+    public String getSample() {
+        return sample;
+    }
+
+    @Override
+    public String getContig() {
+        return contig;
+    }
+
+    @Override
+    public int getStart() {
+        return position;
+    }
+
+    @Override
+    public int getEnd() {
+        return position + 1;
```

Comment:
The `+1` is for `getEnd()` only sometimes, which isn't used for encoding. But I now realize this is actually incorrect since GATK intervals are closed.